tests/lib/feature_dict.py

```diff
@@ -0,0 +1,65 @@
+
+from enum import Enum
+from typing import TypedDict
+
```

Comment:
Nitpick, could you add a module doctring that says those dictionaries are used to clarify some of the deserialized json data passed around?